pkg/sensors/exec/threads_test.go

```diff
@@ -98,7 +98,12 @@ func TestMatchCloneThreadsIDs(t *testing.T) {
 	}
 
 	option.Config.HubbleLib = tus.Conf().TetragonLib
-	tus.LoadSensor(t, base.GetInitialSensor())
+	tus.LoadInitialSensor(t)
+
+	if err := procevents.GetRunningProcs(); err != nil {
```

Comment:
Do we need that here? It seems that `tus.LoadInitialSensor(t)` also calls `procevents.GetRunningProcs()`.

Reply:
ugh, leftover, will remove, thanks